products/views.py

```diff
@@ -32,12 +32,16 @@ def get_list(products):
 class ProductListView(View):
     def get(self, request, category_id=1000):
         try:
+            keyword = request.GET.get('search', '')
+
             category = Category.objects.get(id=category_id)
             products = Product.objects.all()
 
             if category_id != 1000:
                 products = Product.objects.filter(category_id=category_id)
             
+            products = products.filter(name__contains=keyword)
```

Comment:
```suggestion
            products = Product.objects.filter(name__contains=keyword)
```
38번 line의 코드를 지우고 다음과 같이 수정하여도 똑같이 동작 할 것 같습니다!

Reply:
감사합니다!